ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -774,32 +792,57 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)((spinnerPro
     }
   }, [code]);
 
-  // The requiredModules represent those which are required for RT access: RT modules + DUCC
-  // The activeModule must be one of the requiredModules
+  // At any given time, at most two modules will be clickable:
+  //  1. The active module, which we visually direct the user to with a CTA
+  //  2. The next required module, which may diverge when the active module is optional.
+  // This configuration allows the user to skip the optional CT section.
   const [activeModule, setActiveModule] = useState(null);
+  const [clickableModules, setClickableModules] = useState([]);
+
+  const getNextActive = (modules: AccessModule[]) => getActiveModule(getEligibleModules(modules, profile), profile);
+  const nextRequired = getNextActive(requiredModules);
```

Comment:
This was OK because all of these variables are derived from Profile, but this is a bit convoluted to follow. I switched this to make the dependencies clearer.